src/wrapper/jnienv.rs

```diff
@@ -1634,15 +1634,15 @@ impl<'a> JNIEnv<'a> {
 
     /// Get a static field without checking the provided type against the actual
     /// field.
-    pub fn get_static_field_unchecked<T, U>(
+    pub fn get_static_field_unchecked<'c, 'f, T, U>(
         &self,
         class: T,
         field: U,
         ty: JavaType,
     ) -> Result<JValue<'a>>
     where
-        T: Desc<'a, JClass<'a>>,
-        U: Desc<'a, JStaticFieldID<'a>>,
+        T: Desc<'a, JClass<'c>>,
```

Comment:
Does such modification make sense in all the other usages in JNIEnv, where `Desc<'a, JClass/Field/MethodId<'a>>` is accepted, so that they are compatible with global refs  and field/method ids (which have distinct from JNIEnv lifetimes and are allowed to outlive JNIEnv)?

Reply:
Yes, I believe they are also candidates for similar changes. Should they be included in this PR?